src/api/app/models/attrib_value.rb

```diff
@@ -24,6 +25,10 @@ def default_value
       ''
     end
   end
+
+  def normalize_newlines
+    value.gsub!("\r\n", "\n")
```

Comment:
Why does this depend on the newlines?

Reply:
Because I normalize newlines: https://en.wikipedia.org/wiki/Newline

Reply:
I think what you really want to do is `value.gsub!("\r\n?", "\n")` because you want to convert any of the standard line endings (`\n`, `\r\n`, `\r`) to `\n`, don't you? :thinking: I think that is also what @bgeuken means.

But there is IMO a better approach: using `String#encode` with the `universal_newline` option which is more readable and abstract us of how this happen (the regular expression needed)

Reply:
I was actually looking for such a function and couldn't find any - and stackoverflow suggested gsub